extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -59,9 +55,46 @@ sub should_send {
 }
 
 sub send {
+    my ( $self, $message ) = @_;
+
     my $logger = Bugzilla->push_ext->logger;
 
-    $logger->info('AUDIT');
+    my $data = $message->payload_decoded;
+    my $bug_data = $self->_get_bug_data($data) || return 0;
+    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+
+    if(!is_public($bug)) {
```

Comment:
We'd spoken about this earlier but would it make more sense to move this check to the `should_send`?  There doesn't seem to be a point to getting this far if the bug isn't public.